Modules/Utils/Sources/PocketCastsUtils/Feature Flags/FeatureFlag.swift

```diff
@@ -95,6 +95,8 @@ public enum FeatureFlag: String, CaseIterable {
             "cache_playing_episode"
         case .newSettingsStorage:
             "new_settings_storage"
+        case .settingsSync:
+            "settings_sync"
```

Comment:
I think for these two we can do:

```suggestion
        case .newSettingsStorage:
           BuildEnvironment.current != .appStore ? "new_settings_storage" : nil
        case .settingsSync:
            BuildEnvironment.current != .appStore ? "settings_sync" : nil
```

This way any remote feature flag change will only affect the non-App Store builds. Wdyt?

I know this is becoming a bit complex but I think it will keep things safer without us having to change code after merging that.

Reply:
Good point. I combine these checks into a single bool so they don't deviate for now. We can always override individually if needed.